storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -370,7 +368,7 @@ static bool check_server_version(unsigned long version_number,
   }
 
   version_supported =
-      version_supported || (version_number > 80000 && version_number < 90000);
+      version_supported || (version_number > 80000 && version_number <= 80099);
```

Comment:
what about this check block higher versions like 8.1+ and 9.X and 10.X and so on? Separate PR?

Reply:
Ah, right. We don't need another major version check.